R/globals.R

```diff
@@ -1,4 +1,4 @@
 # Undefined global functions or variables
 
-utils::globalVariables(c("EIN", "FormType", "ObjectId", "OrganizationName", "ScheduleO", "ScheduleR", "idx", "path", "re", "year", "%or%", "at_least", "between", "digit", "end", "except_some_of", "group", "maybe", "non_space", "non_spaces", "one_or_more", "start", "zero_or_more", "idx"))
+utils::globalVariables(c("EIN", "FormType", "ObjectId", "OrganizationName", "ScheduleO", "ScheduleR", "idx", "path", "re", "year", "%or%", "at_least", "between", "digit", "end", "except_some_of", "group", "maybe", "non_space", "non_spaces", "one_or_more", "start", "zero_or_more", "idx", "irs_fields","category","package_variable","val"))
```

Comment:
Also, add `'assign(glue("idx_{year}")', "import_idx(year), envir = .GlobalEnv)"` for the global variables. I added them as last-minute changes for the last merged commit. If you didn't add this, merging this PR would cause a conflict with the main branch.